src/SMSManager.php

```diff
@@ -15,23 +16,19 @@ final class SMSManager
     private $smsServiceProviderDriver;
 
     /**
-     * @param string $smsServiceProviderDriver
-     * @return SMSManager
      * @throws UndefinedDriver
-     * @throws \ReflectionException
+     * @throws ReflectionException
      */
-    public function via (string $smsServiceProviderDriver): SMSManager
+    public function __construct()
     {
-        $this->smsServiceProviderDriver = $this->getDriverInstance($smsServiceProviderDriver);
-
-        return $this;
+        $this->smsServiceProviderDriver = $this->getDriverInstance(config('resala.default_driver'));
     }
```

Comment:
What do you think about using the constructor to use the default driver as you did here, but with keeping the _via_ function to have the option of selecting a driver other than the default
I think this will be useful in case we are using multiple sms drivers in the same project.